Evaluation/Input/Content/Section2.1_Modeling_Strategy.md

```
@@ -0,0 +1,20 @@
+
+Unknown parameters were identified using the Parameter Identification module provided in PK-Sim®. Structural model selection was mainly guided by visual inspection of the resulting description of data and biological plausibility.
+
+Once the appropriate structural model was identified, additional parameters for tablet formulations were identified. 
+
+The model was then verified by simulating:
+
+- IV administration
+- Oral administration in fasted state
+- Oral administration in fed state
+
+Details about input data (physicochemical, *in vitro* and clinical) can be found in  [Section 2.2](#methods-data).
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)